src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/Runtime/TypeInfos/NativeFormat/NativeFormatRuntimeNamedTypeInfo.cs

```diff
@@ -48,9 +48,10 @@ public sealed override bool IsByRefLike
                     return Internal.Runtime.Augments.RuntimeAugments.IsByRefLike(typeHandle);
 
                 // Otherwise fall back to attributes
+                string[]? isByRefLikeAttributeNamespaceParts = null;
                 foreach (CustomAttributeHandle cah in _typeDefinition.CustomAttributes)
                 {
-                    if (cah.IsCustomAttributeOfType(_reader, "System.Runtime.CompilerServices", "IsByRefLikeAttribute"))
+                    if (cah.IsCustomAttributeOfType(_reader, isByRefLikeAttributeNamespaceParts ??= ["System", "Runtime", "CompilerServices"], "IsByRefLikeAttribute"))
```

Comment:
Can it be passed as a span instead of array? I'm not sure if there's any limitation for AOT here.

Reply:
Yup, I didn't realize Roslyn would be able to elide the array allocation. Looks better now.